x-pack/plugins/ml/server/models/data_recognizer/modules/security_windows/ml/v3_windows_anomalous_script.json

```diff
@@ -0,0 +1,53 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "Looks for unusual powershell scripts that may indicate execution of malware, or persistence mechanisms.",
+  "groups": [
+    "endpoint",
+    "event-log",
+    "process",
+    "windows",
+    "winlogbeat",
+    "powershell"
+  ],
+  "analysis_config": {
+    "bucket_span": "15m",
+    "detectors": [
+      {
+        "detector_description": "Security: Windows - Detects high information content in powershell.file.script_block_text values.",
```

Comment:
You made the edit to the `detector_description`, but it should be in the job `description` field above. The convention used for all the other jobs is to start all the job descriptions with `Security: Windows - `

Reply:
fixed